android/java/org/chromium/chrome/browser/crypto_wallet/fragments/SwapBottomSheetDialogFragment.java

```diff
@@ -67,6 +65,7 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         try {
             mWalletModel = BraveActivity.getBraveActivity().getWalletModel();
+            registerKeyringObserver(mWalletModel.getKeyringModel());
         } catch (BraveActivity.BraveActivityNotFoundException e) {
             Log.e(TAG, "onCreate ", e);
             e.printStackTrace();
```

Comment:
`e.printStackTrace()` can be removed as it will be already printed from calling `Log.e`.